CHANGELOG.md

```diff
@@ -51,6 +51,8 @@ list only contains the Pylint checkers enabled by default in PythonTA.
 - Moved tests related to `__main__.py` from `test_check.py` to `test_main.py`
 - Added more unit tests to `test_main.py` to increase coverage of `__main__.py` to 100%
 - Updated `README.md` to reflect updated folder structure
+- Moved tests related to `snapshot.py` out of `test_accumulation_table.py` and into new module `test_snapshot.py`
+- Updated GitHub Action tests to ignore `test_accumulation_table.py` and add verbose output for debug testing
```

Comment:
The first part ("ignore `test_accumulation_table.py`) is not the correct behaviour. Those tests should still be executed in the second `pytest` run, but ignored in the first run.

You can keep the "add verbose output" part though, that's good.